test/school_house_web/controllers/live/conference_live/index_test.exs

```diff
@@ -7,12 +7,12 @@ defmodule SchoolHouseWeb.ConferenceLiveIndexTest do
 
   describe "mount/3" do
     test "can mount live view", %{conn: conn} do
-      {:ok, view, html} = live_isolated(conn, SchoolHouseWeb.ConferenceLive.Index, session: %{})
+      {:ok, _view, html} = live_isolated(conn, SchoolHouseWeb.ConferenceLive.Index, session: %{})
```

Comment:
Thanks for catching these unused variables! I wonder why dialyzer and credo missed them.

Reply:
I think we'd need to run credo and dialyzer with `MIX_ENV=test`. We should give that a try and if it works I'm okay with updating the workflow